apps/backoffice-v2/src/common/components/organisms/Sidebar/Sidebar.GroupLabel.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { Slot } from '@radix-ui/react-slot';
+import { ctw } from '@ballerine/ui';
+
+export const SidebarGroupLabel = React.forwardRef<
+  HTMLDivElement,
+  React.ComponentProps<'div'> & { asChild?: boolean }
+>(({ className, asChild = false, ...props }, ref) => {
+  const Comp = asChild ? Slot : 'div';
+
+  return (
+    <Comp
+      ref={ref}
+      data-sidebar="group-label"
+      className={ctw(
+        'text-sidebar-foreground/70 ring-sidebar-ring [&>svg]:size-4 flex h-8 shrink-0 items-center rounded-md px-2 text-xs font-medium outline-none transition-[margin,opa] duration-200 ease-linear focus-visible:ring-2 [&>svg]:shrink-0',
```

Comment:
transition-[margin,opa] looks like it might be broken and supposed to be transition-[margin,opacity]

Reply:
Nice catch!